src/API/OnboardingTasks.php

```diff
@@ -686,14 +742,10 @@ public function dismiss_task( $request ) {
 
 		$is_dismissable = false;
 
-		foreach ( OnboardingTasksFeature::get_task_lists() as $task_list ) {
-			foreach ( $task_list['tasks'] as $task ) {
-				// @todo Use the reusable methods to get the task introduced in https://github.com/woocommerce/woocommerce-admin/pull/7539
-				if ( $id === $task['id'] && isset( $task['isDismissable'] ) && $task['isDismissable'] ) {
-					$is_dismissable = true;
-					break;
-				}
-			}
+		$dismiss_task = OnboardingTasksFeature::get_task_by_id( $id );
```

Comment:
Thanks for using your new util here!  Nitpicky, but this variable sounds like a verb.  Maybe just `$task`?

```suggestion
		$task = OnboardingTasksFeature::get_task_by_id( $id );
```